ethereum/contracts/Bridge.sol

```diff
@@ -223,42 +259,29 @@ contract Bridge is AccessControl {
     // ************    ADMIN-LEVEL ACCESS METHODS   *************
 
 
-    function inflate(uint256 amount)
+    function mint(uint256 amount)
         public
         onlyOwner
     {
-        inflationAmount = inflationAmount.add(amount);
-
-        // NOTE(pb): This needs to be done, so inflationary amount of tokens from counterpart blockchain corresponding
-        //  to `effectiveAmountAccrued` *ORIGINALLY* transferred in to that blockchain (= before inflation) can be
-        //  transferred back to the *source*(=Ethereum) blockchain.
-        //  Please keep in mind that this action **SHALL** be preceded with **MINTING** of the `amount`
-        //  of ERC20 FET tokens.
-         effectiveAmountAccrued = effectiveAmountAccrued.add(amount);
-        emit Inflate(amount);
+        // NOTE(pb): The `effectiveAmountAccrued` shall be adjusted by deflation amount.
+        effectiveAmountAccrued = effectiveAmountAccrued.add(amount);
+        token.mint(address(this), amount);
 
         // NOTE(pb): We should think if alignment of cap is actually right thing to do here.
         _setCap(cap.add(amount));
     }
 
 
-    function deflate(uint256 amount)
+    function burn(uint256 amount)
```

Comment:
Similarly, a burn does not affect the cap so no need to mess with it in the function.

Reply:
Resolved in following commits - please see [my comment](https://github.com/fetchai/fetch-ethereum-bridge-v1/pull/2#issuecomment-770809906) sumarising the changes reflecting requirements which eraised during by this discussion.